frontend/components/RechargeOption/components/SheetItem/index.jsx

```diff
@@ -42,9 +43,11 @@ const SheetItem = ({
     onClick: handleItemClick,
   });
 
+  const label = singularizeOrderUnit(item, intervalUnit)
+
   return (
     <button {...buildProps()} aria-selected={selected} role="option">
```

Comment:
I know this wasn't part of your changes but the linter is complaining that button should have a type attribute. Its value should be "button"

Reply:
fixed